package.xml

```diff
@@ -10,10 +10,10 @@
         <email>sammyk@php.net</email>
         <active>yes</active>
     </lead>
-    <date>2019-07-02</date>
+    <date>${date}</date>
     <version>
-        <release>0.29.0</release>
-        <api>0.29.0</api>
+        <release>${version}</release>
+        <api>${version}</api>
     </version>
     <stability>
         <release>stable</release>
```

Comment:
Can we update also `<notes></notes>` a few lines down  this line please? :)

Reply:
@morrisonlevi just a reminder about this suggestion of replacing the content of `<note></note>` with `${note}`.

Reply:
@morrisonlevi sorry I meant `<notes></notes>`